src/Symfony/Component/Form/Extension/Core/Type/DateType.php

```diff
@@ -49,7 +49,12 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $dateFormat = \is_int($options['format']) ? $options['format'] : self::DEFAULT_FORMAT;
         $timeFormat = \IntlDateFormatter::NONE;
-        $calendar = $options['calendar'] ?? \IntlDateFormatter::GREGORIAN;
+        if (isset($options['calendar'])) {
+            $calendar = $options['calendar'];
+        } else {
+            $calendar = new \IntlGregorianCalendar();
```

Comment:
this adds a strict dependency to ext-intl for the default configuration because `symfony/polyfill-intl-icu` does not provide a polyfill for `IntlGregorianCalendar`

Reply:
btw, this also changes the existing behavior for forms for the default setup, which might cause other issues. Maybe you should instead solve your case by configuring a `calendar` option (either explicitly when using the DateType or project-wide by registering a FormTypeExtension replacing the default value of the `calendar` option)